Dockerfiles/midstream/Dockerfile

```diff
@@ -13,6 +13,8 @@ RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm6
     mkdir /project/output && \
     dnf install --setopt=install_weak_deps=False -y git-core ansible skopeo && \
     dnf clean all
+ADD ./entrypoint.sh /entrypoint.sh
```

Comment:
Can you move this `ADD` instruction to be above line 6 and then incorporate the `RUN chmod g+x /entrypoint.sh && chmod g+w /etc/passwd` commands into the `RUN` instruction at line 6?

Reply:
Sure I can do that.